core/src/com/mygdx/game/core/ecs/component/PlayerToken.java

```diff
@@ -0,0 +1,19 @@
+package com.mygdx.game.core.ecs.component;
+
+import com.artemis.PooledComponent;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+@Data
+@NoArgsConstructor
+public class PlayerToken extends PooledComponent {
+
+  @NonNull
+  private String token; // TODO change to static string as in Name
```

Comment:
w sumie to ten token w ogóle nie jest potrzebny jako komponent, bo po co to komukolwiek, może oprócz serwera, ale na razie nic z tym nie robi

tutaj powinien być bardziej name tylko, albo jakiś inny identyfikator gracza, który nie musiałby być sekretem, dzięki, któremu reszta graczy będzie mogła identyfikować innych,

jakby chciało się dodać funkcjonalność reconnectowania jak kogoś wywali (a to powinno być), to token/hasło mogłoby służyć do autentykacji, bo klient wysłałby ten sam token i serwer zobaczyłby, że mógłby "odmrozić" dane rozłączonego klienta i tylko podmienić socketa i działałby ok

Reply:
"token" używam dlatego bo zakładam że nick może się powtórzyć